<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1056</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="styleSheet">
           <string notr="true">QTextBrowser {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:black;
}


</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="fontComboBox">
            <property name="styleSheet">
             <string notr="true">QComboBox {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QComboBox:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sizeComboBox">
            <property name="styleSheet">
             <string notr="true">QComboBox {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QComboBox:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">QPushButton {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QPushButton:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
              </property>
              <property name="text">
               <string>加粗</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="underlineToolBtn">
              <property name="styleSheet">
               <string notr="true">QPushButton {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QPushButton:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
              </property>
              <property name="text">
               <string>下划线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sendimagebutton">
              <property name="styleSheet">
               <string notr="true">QPushButton {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QPushButton:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
              </property>
              <property name="text">
               <string>发送图片</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sendToolBtn">
              <property name="styleSheet">
               <string notr="true">QPushButton {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QPushButton:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
              </property>
              <property name="text">
               <string>发送文件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveToolBtn">
              <property name="styleSheet">
               <string notr="true">QPushButton {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QPushButton:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
              </property>
              <property name="text">
               <string>下载文档</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearToolBtn">
              <property name="styleSheet">
               <string notr="true">QPushButton {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QPushButton:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
              </property>
              <property name="text">
               <string>清空聊天记录</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:black;
}


</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QPushButton:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
          </property>
          <property name="text">
           <string>发送数据</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;当前人数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="indent">
           <number>-2</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="exitpushButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
QPushButton:hover {
border: 0.5px solid white;
	border-color: rgb(160,160,160);
border-radius: 6px;
background-color:rgba(84,84,84,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}

</string>
          </property>
          <property name="text">
           <string>推出聊天</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
border: 0.5px solid white;
border-color: rgb(160,160,160);
border-radius: 6px;
background-color: rgba(160,160,160,50);
min-width: 80px;
font-family:&quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}


</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Qlabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>485</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Qlabel</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearToolBtn</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>763</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
